lib/clarkson-core-deprecated.php

```diff
@@ -21,6 +21,35 @@ public function __construct(){
         }
     }
 
+    public function get_theme_objects(){
+      $objects = array();
```

Comment:
your tabs / spaces are different... please fix to be inline with the rest of the files